tests/scientific/test_extracellular_stimulation.py

```diff
@@ -0,0 +1,95 @@
+from pathlib import Path
+
+import numpy as np
+import numpy.testing as npt
+import pytest
+
+SIM_DIR = Path(__file__).parent.parent.absolute() / "simulations"
+
+
+@pytest.mark.parametrize(
+    "create_tmp_simulation_config_file",
+    [
+        {
+            "src_dir": str(SIM_DIR / "usecase3"),
+            "simconfig_file": "simulation_sonata.json",
+            "extra_config": {
+                "target_simulator": "NEURON",
+                "inputs": {
+                    "ex_efields": {
+                        "input_type": "extracellular_stimulation",
+                        "module": "spatially_uniform_e_field",
+                        "delay": 0,
+                        "duration": 10,
+                        "node_set": "Mosaic",
+                        "fields": [
+                            {"Ex": 50, "Ey": -25, "Ez": 75, "frequency": 100},
+                            {"Ex": 100, "Ey": -50, "Ez": 50, "frequency": 0},
```

Comment:
nitpick: what about a 3rd frequency, very low like 0.001? So we can check if the results are basically the same as 0 and there are not jumps in the results. Just an idea, if you want